app/views/comments/_comment.html.erb

```diff
@@ -124,7 +124,7 @@ class="comment <%= comment.current_vote ? (comment.current_vote[:vote] == 1 ?
           <% if comment.flags > 0 &&
           comment.show_score_to_user?(@user) &&
           (comment.user_id == @user.try(:id) || @user.try("is_moderator?")) %>
-            | <%= comment.vote_summary_for_user(@user).downcase %>
+            <%= "| #{comment.vote_summary_for_user(@user).downcase}" if comment.vote_summary_for_user(@user) %>
```

Comment:
Thank you @hmadison for helping me make this patch more concise.

Here the side-effect is that we invoke vote_summary_for_user twice--if the template language supports local variables I can use that to avoid doing that, here. (Or, clearly, one could memoize the function call.)

Reply:
side effect addressed in the latest patch set.